modules/monitoring-setting-the-body-size-limit-for-metrics-scraping.adoc

```diff
@@ -52,10 +52,4 @@ Valid numeric values use the Prometheus data size format: B (bytes), KB (kilobyt
 The default value is `0`, which specifies no limit.
 You can also set the value to `automatic` to calculate the limit automatically based on cluster capacity.
 
-. Save the file to apply the changes automatically.
-+
-[WARNING]
-====
-When you save changes to a `cluster-monitoring-config` config map, the pods and other resources in the `openshift-monitoring` project might be redeployed.
-The running monitoring processes in that project might also restart.
-====
+. Save the file to apply the changes. The new configuration is applied automatically.
```

Comment:
@juzhao  When I tested this, this change did not trigger pods to redeploy so I am not including the sentence about the redeployment here.

Reply:
yes, not restarted, configuration loaded to prometheus
```
$ oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- cat /etc/prometheus/config_out/prometheus.env.yaml | grep body_size_limit | uniq
  body_size_limit: 40MB
```